gpt/gpt/gpt.py

```diff
@@ -0,0 +1,214 @@
+"""Welcome to Pynecone! This app is a demonstration of OpenAI's GPT."""
+import pynecone as pc
+from .helpers import navbar
+import openai
+
+openai.api_key = "YOUR_API_KEY"
+
+
+class User(pc.Model, table=True):
+    username: str
+    password: str
+
+
+class Question(pc.Model, table=True):
+    username: str
+    prompt: str
+    question: str
+
+
+class State(pc.State):
+    """The app state."""
+
+    username: str
+    password: str
+    logged_in: bool = False
+
+    prompt: str
+    result: str
+
+    @pc.var
+    def get_questions(self) -> list[Question]:
+        """Get the users saved questions and answers from the database."""
+        with pc.session() as session:
+            if self.logged_in:
+                return session.exec(Question.select.where(Question.username == self.username)).all()
```

Comment:
I *think* this can be simplified as `session.query(Question).where(Question.username == self.username).all()`

Reply:
Or maybe `filter` instead of `where`